### PR TITLE
Cumulus 1558 2 TF configs

### PR DESCRIPTION
**Pull request by laurenfrederick**
**Summary:** 

Terraform API port configs and configuration for private archive API gateway

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: tf-modules/archive/api.tf [hunk 1]
```diff
@@ -82,6 +82,10 @@ resource "aws_api_gateway_rest_api" "api" {
   lifecycle {
     ignore_changes = [policy]
   }
+
+  endpoint_configuration {
```
Comment (markdboyd) [inline tf-modules/archive/api.tf]: Would we need this for our distribution API as well? Or is that separate bc it's handled by TEA?

File: example/cumulus-tf/variables.tf [hunk 2]
```diff
@@ -105,3 +105,13 @@ variable "log_destination_arn" {
   default     = null
   description = "Remote kinesis/destination arn for delivering logs. Requires log_api_gateway_to_cloudwatch set to true."
 }
+
+variable "archive_api_port" {
```
Comment (markdboyd) [inline example/cumulus-tf/variables.tf]: Where is this ultimately used?

File: tf-modules/cumulus/archive.tf [hunk 3]
```diff
@@ -43,6 +43,7 @@ module "archive" {
   dynamo_tables = var.dynamo_tables
 
   api_port = var.archive_api_port
```
Comment (laurenfrederick) [inline tf-modules/cumulus/archive.tf]: @markdboyd here!

File: example/cumulus-tf/variables.tf (hunk 2 above)
Comment (laurenfrederick) [reply example/cumulus-tf/variables.tf]: I tagged you on the line

File: tf-modules/archive/api.tf (hunk 1 above)
Comment (laurenfrederick) [reply tf-modules/archive/api.tf]: Yep the API Gateway is deployed by TEA. I should check though how we are configuring TEA.

Comment (laurenfrederick) [reply tf-modules/archive/api.tf]: Looks like we are good there

**Pull request merged**